src/components/AudioPlayer/Player.js

```diff
@@ -251,8 +257,16 @@ class AudioPlayer extends Component {
         loading: true
       }))
     }
+    this.setTime = (time = 0) => {
+      if (this.audio.currentTime !== time) {
+        this.audio.currentTime = time
+        this.updateProgress()
+      }
+    }
     this.onCanPlay = () => {
+      !this.state.initialized && this.setTime(this.props.startSeconds)
```

Comment:
also add this to `onLoadedMetaData`, iOS apparently does not fire `onCanPlay`.

Reply:
I've created `initStartTime` now which is called both from `onCanPlay`, `onLoadedMetaData`, `componentDidMount` and `componentDidUpdate`. The first two event handlers may not even fire in some cases (e.g. when opening a new tab on desktop with the same article).